pkg/discovery/worker/metrics_collector.go

```diff
@@ -123,7 +129,7 @@ func (collector *MetricsCollector) CollectPodMetrics() PodMetricsByNodeAndQuota
 
 				var quotaComputeCap float64
 				quotaComputeCap, err := getQuotaComputeCapacity(quota, computeType)
-				if err == nil && quotaComputeCap < podCpuCap {
+				if err == nil && quotaComputeCap != repository.DEFAULT_METRIC_VALUE && quotaComputeCap < podCpuCap {
```

Comment:
It will be easier for debugging if you can log the error, and the other conditions.